packs/caasperli/README.md

```diff
@@ -0,0 +1,30 @@
+# Caasperli Nomad Pack
+
+This is a pack for [Caasperli](https://github.com/adfinis-sygroup/potz-holzoepfel-und-zipfelchape), a simple container used for demonstration purposes.
```

Comment:
@in0rdr the only reference I could find that defines what Cassperli is was this blog post https://adfinis.com/en/blog/how-to-deploy-suse/. The source for the application doesn't appear anywhere in the adfinis-sysgroup's GitHub organization either. Without being able to review the source and verify that the license allows for distribution, we can't accept this PR.